src/ui/pages/SendForm/SendForm.tsx

```diff
@@ -594,6 +594,7 @@ function SendFormComponent() {
                     chain={chain}
                     sendView={sendView}
                     paymasterEligible={paymasterEligible}
+                    eligibilityQuery={eligibilityQuery}
```

Comment:
Can we lose eligibilityQuery.data here because of spread operator above?

Reply:
It looks like we missed interpretations for send form because of this

Reply:
good catch, thank you!